compiler/testData/diagnostics/tests/operatorsOverloading/assignOperatorOverload.kt

```diff
@@ -0,0 +1,76 @@
+// FIR_IDENTICAL
+// !DIAGNOSTICS: -UNUSED_PARAMETER
+// !LANGUAGE:+AssignOperatorOverloadForJvm
```

Comment:
I don't see any tests for feature switched off

Reply:
There is [compiler/testData/diagnostics/tests/operatorsOverloading/assignOperatorOverloadDisabled.kt](https://github.com/JetBrains/kotlin/pull/4890/files#diff-20b915b704de1013df4264283f347d7a1c8e3905a5d8f10f5a6f6c8468e272d5). Let me know if I should rename it.

Reply:
Renamed to compiler/testData/diagnostics/tests/operatorsOverloading/assignOperatorOverload_disabled.kt so it's easier to spot.